tracer/src/Datadog.Trace/TracerManager.cs

```diff
@@ -354,6 +354,9 @@ private static async Task WriteDiagnosticLog(TracerManager instance)
                     writer.WritePropertyName("routetemplate_expansion_enabled");
                     writer.WriteValue(instanceSettings.ExpandRouteTemplatesEnabled);
 
+                    writer.WritePropertyName("obfuscation_querystring_regex_timout");
+                    writer.WriteValue(instanceSettings.ObfuscationQueryStringRegexTimeout);
```

Comment:
Could be interesting to add the status of url obfuscation here too, and maybe query string collection, if it's not already logged.

Reply:
fixed in c5c4456c0eec985161bda650c99c0e468c5dc26b